pages/linux/trash.md

```diff
@@ -0,0 +1,23 @@
+# trash
+
+> A CLI trashcan/recycling bin.
+
+- Delete a file (send to trash):
+
+`trash-put {{path/to/file}}`
+
+- List files in trash:
+
+`trash-list`
+
+- Restore file from trash:
+
+`trash-restore`
+
+- Empty trash:
+
+`trash-empty`
+
+- Empty trash, keeping files trashed less than {{X}} days ago:
+
+`trash-empty {{X}}`
```

Comment:
Perhaps use a concrete example, such as `10`:
```suggestion
`trash-empty {{10}}`
```
And then update the description to:
`Empty trash, keeping files trashed less than 10 days ago:`

Reply:
fixed